.github/workflows/lean-release-tag.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - 'main'
+      - 'master'
```

Comment:
```suggestion
```
There is no `master` branch on this repo. Should we leave this line in anyway? My instinct is "no".